docker/prepare-environment.sh

```diff
@@ -15,3 +19,5 @@ fi
 
 # Enable ding PHP configuration.
 phpenmod ding
+
+mkdir -p /root/.ssh && touch /root/.ssh/known_hosts && ssh-keyscan -H github.com >> /root/.ssh/known_hosts && chmod 600 /root/.ssh/known_hosts
```

Comment:
This seems no better than just configuring your ssh client to ignore host ket checking.

How about just hard coding the currently known host keys? https://help.github.com/articles/github-s-ssh-key-fingerprints/

I couldn't find a Github API giving the host keys ... that would have been even better.